### DISCUSSION TITLE
Scroll in docs

### DISCUSSION THREAD

**Discussion by ghost** [helix-editor/helix#2344] [Q&A] [answered]
How can one scroll documentation opened with `Space + k`? I tried `Ctrl+u` and `Ctrl+d` but none of them work. Also can't find anything in the [Helix docs keybindings section](https://docs.helix-editor.com/keymap.html).

**Comment by kirawi** [comment, marked as answer]
`ctrl + u` and `ctrl + d` should work, so maybe there isn't any documentation to scroll?

**Comment by getreu** [comment]
I works for me with `ctrl + u` and `ctrl + d`: helix 22.05-dev (2c60798b)
